src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    categories: [],
+    status: 'Under construction',
+  },
+  reducers: {
+    checkStatus: (state) => state,
```

Comment:
- [ ] You did a great job setting up the reducers. :+1: I don't think this particular reducer will return the string `Under construction`, it's going to return an object containing an array of categories and a status string. Kindly make it so that it always returns the string `Under construction`. 🤝️

**Example**
```
checkStatus: (state) => state.status
```